pkgs/servers/sql/cockroachdb/default.nix

```diff
@@ -1,70 +1,12 @@
-{ lib, stdenv, buildGoPackage, fetchurl
-, cmake, xz, which, autoconf
-, ncurses6, libedit, libunwind
-, installShellFiles
-, removeReferencesTo, go
-}:
-
-let
-  darwinDeps = [ libunwind libedit ];
-  linuxDeps  = [ ncurses6 ];
-
-  buildInputs = if stdenv.isDarwin then darwinDeps else linuxDeps;
-  nativeBuildInputs = [ installShellFiles cmake xz which autoconf ];
-
-in
-buildGoPackage rec {
-  pname = "cockroach";
-  version = "20.1.8";
-
-  goPackagePath = "github.com/cockroachdb/cockroach";
-
-  src = fetchurl {
-    url = "https://binaries.cockroachdb.com/cockroach-v${version}.src.tgz";
-    sha256 = "0mm3hfr778c7djza8gr1clwa8wca4d3ldh9hlg80avw4x664y5zi";
+self: {
+  cockroachdb_20_1 = self.callPackage ./gopkg.nix {};
```

Comment:
How would you feel about removing this completely and only leave 20.2 and upwards?

Reply:
This would have the advantage that it could be called `cockroachdb_20` and `cockroachdb_21`.

Reply:
Ok, we've talked about this. We need the second component.